site/faq.md

```diff
@@ -96,8 +96,13 @@ explanation.
 
 ### Why does Flux need a deploy key?
 
-Flux needs a deploy key to be allowed to push to the version control
-system in order to read and update the manifests.
+There are a number of Flux commands and API calls which will update the git repo in order to
+reflect the state of the cluster after applying the command. For more information about flux
+commands see [the fluxctl docs](./using.md).
+
```

Comment:
Oh yes, updating this bit helps too :-) There's two bits to the answer: one is why we need a deploy key (to get access to the git repo), and the other is why Flux needs both read and write access. You could cover both by explaining the need for read access to sync, and the need for write access to update git when asked.

Secondarily -- "reflect the state of the cluster after applying the command" doesn't follow, because it suggests that commands take what's in the cluster and put that in git. In general, the commands update manifests according to the user's request -- "update the deployment's image to be foo:v1" -- rather than mirroring information from the cluster.

Reply:
If it were a public repo, you wouldn't need the key for read access and I can hardly imagine that people frequently get confused about Flux needing a key to read from private repos. For me, the main source of confusion is why Flux needs write access.

Reply:
That's a good point -- what you've written above explains it well.